### ISSUE TITLE
skin_variations missing for new kiss/heart emoji combinations

### ISSUE THREAD

**Issue by jwheare** [iamcal/emoji-data#195]
Post 13.1

`COUPLE WITH HEART` has `skin_variations` set in the json but e.g. `COUPLE WITH HEART: WOMAN, MAN` doesn't, nor does any other gendered version. As a result, only the non gender specific version has skin tones.

This also effects `KISS`

**Comment by jwheare** [comment]
Adding these to build/data_emoji_names_v13_1.txt and rebuilding should fix it:

```
{MAN/WOMAN}{SKIN!}-200d-2764-fe0f-200d-1f48b-200d-{MAN/WOMAN}{SKIN2};{M/W}-kiss-{M/W}
1F469{SKIN!}-200d-2764-fe0f-200d-1f48b-200d-1F468{SKIN2};woman-kiss-man
{MAN/WOMAN}{SKIN!}-200d-2764-fe0f-200d-{MAN/WOMAN}{SKIN2};{M/W}-heart-{M/W}
1F469{SKIN!}-200d-2764-fe0f-200d-1F468{SKIN2};woman-heart-man
```

I don't have the latest apple font to hand to test this fully with the images, but it fixes the json at least.

**Comment by iamcal** [comment]
good catch - there's something missing in the map builder because it should be flagging these as missing, so i'll patch that first. i suspect it's only checking base emoji and not skin-tone variations

**Comment by iamcal** [comment]
yeah - after commit 0813c5b58 i can see we're missing 190 emoji. going to need to re-extract images and rebuild everything, which takes a few hours

**Comment by iamcal** [comment]
I have uncovered so many bugs 😂

**Comment by iamcal** [comment]
there are now 190 new variations added to the data file on trunk. i need to do a full rebuild and re-optimize of the sheets

**Comment by iamcal** [comment]
Fixed in 7.0.1
